css/cssom-view/matchMedia.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<meta name="flags" content="dom">
+<title>CSS Test: CSSOM View matchMedia and MediaQueryList</title>
+<link rel="author" title="Rune Lillesveen" href="mailto:rune@opera.com">
+<link rel="help" href="http://www.w3.org/TR/cssom-view/#dom-window-matchmedia">
+<link rel="help" href="http://www.w3.org/TR/cssom-view/#the-mediaquerylist-interface">
+<link rel="help" href="http://www.w3.org/TR/cssom-1/#serializing-media-queries">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="resources/matchMedia.js"></script>
+<div id="log"></div>
+<script>
+"use strict";
+
+test(() => {
+    assert_equals(
+        typeof window.matchMedia,
+        "function",
+        "FATAL ERROR: The window.matchMedia function is not present. The rest of the testsuite will fail to run."
+    );
+}, "window.matchMedia is a function");
+
+test(() => {
+    assert_inherits(window, "matchMedia");
```

Comment:
This test is bogus, since window is special. IDL attributes are properties on the instance for globals.

https://heycam.github.io/webidl/#es-attributes

This should be tested by idlharness, so this subtest can be removed.

/css/cssom-view/idlharness.html

https://github.com/web-platform-tests/wpt/blob/e501cacc172f9ee2485d27e8d3be154425be9cb8/resources/idlharness.js#L2413-L2419

Reply:
TIL, thank you, removed in f58e6d5.